bedrock/firefox/views.py

```diff
@@ -343,26 +343,35 @@ def show_57_firstrun(version):
     return version >= Version('57.0')
 
 
-def show_40_firstrun(version):
+def show_57_dev_firstrun(version):
+    version = version[:-2]
```

Comment:
why only stripping the last two characters? Will it definitely be something like "40.1"? Could be "40.0.1" right? Or is that to get rid of the "a2" or whatever it is?

Reply:
Hmm, this isn't actually part of this PR, b ut I think you're right yes it is to remove the `a2` part from the Dev Edition version number.

Reply:
Ah! Yes. I missed that the diff was just showing the line being moved. Carry on :)